### ISSUE TITLE
Day 40: Tasks Application

### ISSUE THREAD

**Issue by mchirico** [mchirico/zDaily#44]
[video](https://vimeo.com/482017045)

# Designing Tasks Application

Okay, involves kind of a lot... there is potential for a lot of creativity; but, we 
have to install some basic infrastructure first. 

We want to get a working prototype.  That way, we
can test ideas, in the real world.

- [x] Go practice program (as always), maps, closures... make it personal
- [x] Review design of project
- [x] Download [etcd](https://github.com/etcd-io/etcd)
- [x] Download, modify [github.com/mchirico/go-etcd](https://github.com/mchirico/go-etcd). See video
- [ ] Confirm that you can get [iOSurl](https://github.com/mchirico/iOSurl) running on your phone 
- [x] Talk to Leah about creating icons

![image](https://user-images.githubusercontent.com/755710/99878749-78399480-2bd5-11eb-8e35-c5e6bcd10d7a.png)

**Comment by ZoeChiri** [comment]
confused on iOSurl still

**Comment by mchirico** [comment]
Yeah, there's a lot in iOSurl... we can walk through a simpler example.  It actually does a lot:

- Has Sqlite database
- Segues 
- Background processes
- Http calls
